display_list/effects/dl_image_filter_unittests.cc

```diff
@@ -209,6 +209,16 @@ TEST(DisplayListImageFilter, BlurBounds) {
   TestBounds(filter, input_bounds, expected_output_bounds);
 }
 
+TEST(DisplayListImageFilter, BlurZeroSigma) {
+  std::shared_ptr<DlImageFilter> filter =
+      DlBlurImageFilter::Make(0, 0, DlTileMode::kMirror);
+  ASSERT_EQ(filter, nullptr);
+  filter = DlBlurImageFilter::Make(3, SK_ScalarNaN, DlTileMode::kMirror);
+  ASSERT_EQ(filter, nullptr);
+  filter = DlBlurImageFilter::Make(3, 0, DlTileMode::kMirror);
+  ASSERT_NE(filter, nullptr);
```

Comment:
It couldn't hurt to check

- NaN,NaN
- Nan,3
- 0,3

as well...

Reply:
done